src/test/java/science/icebreaker/device_availability/DeviceAvailabilityServiceTest.java

```diff
@@ -1,14 +1,11 @@
 package science.icebreaker.device_availability;
 
+import org.junit.jupiter.api.*;
```

Comment:
It would be better to list the imports explicitly instead of using a wildcard to avoid confusion and clutter. Maybe it's not that serious when it comes to testing, but it will be rejected by checkstyle.

Reply:
when you autoimport annotations in that package, IDE is doing wildcard automatically after 4 or 5 imports from same package

Reply:
Ah I see. for the time being you can leave it. it should be auto adjusted in the future.